### PR TITLE
Update footer style and navigation tags

### PR DESCRIPTION
**Pull request by JuliaEvseeva**
This PR brings updated styles for the footer.
Also, the order of tags in navigation was changed.

### CODE REVIEW

**Comment by JuliaEvseeva** [comment]
@andreySadomovskiy PTAL

File: _sass/modules/_nav.scss [hunk 1]
```diff
@@ -246,6 +247,7 @@
   a {
```
Comment (andreySadomovskiy) [inline _sass/modules/_nav.scss]: You can use SASS feature here:
```
a {
    color: rgba($white, 0.64);
    
    &.current {
      text-decoration: underline;
      color: $white;
    }
  } 
```
and fix `.current` item styles
```
a {
    color: rgba($white, 0.64);

    &.current {
      text-decoration: underline;
      color: $white;

      .nav-doc-tab {
        color: $white;
      }
    }
  }
```

File: _includes/nav.html [hunk 2]
```diff
@@ -48,11 +48,19 @@
...
         <a href="{{ site.baseurl }}/docs/examples" {% if current[2] == 'examples' %}class='current'{% endif %}><li class="nav-doc-tab">Examples</li></a>
       </ul>
+      <!--<ul class="doc-list">-->
+        <!--<li class="nav-doc-tab"><a href="{{ site.baseurl }}/docs" {% if page.path == 'docs/index.html' %}class='current'{% endif %}>Overview</a></li>-->
+        <!--<li class="nav-doc-tab"><a href="{{ site.baseurl }}/docs/quickstart" {% if current[2] == 'quickstart' %}class='current'{% endif %}>Quick Start</a></li>-->
+        <!--<li class="nav-doc-tab"><a href="{{ site.baseurl }}/docs/guides" {% if current[2] == 'guides' %}class='current'{% endif %}>Guides</a></li>-->
+        <!--<li class="nav-doc-tab"><a href="{{ site.baseurl }}/docs/tutorials" {% if current[2] == 'tutorials' %}class='current'{% endif %}>Tutorials</a></li>-->
+        <!--<li class="nav-doc-tab"><a href="{{ site.baseurl }}/docs/reference" {% if current[2] == 'reference' %}class='current'{% endif %}>Reference</a></li>-->
+        <!--<li class="nav-doc-tab"><a href="{{ site.baseurl }}/docs/examples" {% if current[2] == 'examples' %}class='current'{% endif %}>Examples</a></li>-->
+      <!--</ul>-->
```
Comment (andreySadomovskiy) [inline _includes/nav.html]: Remove this commented code, please.

**Comment by andreySadomovskiy** [comment]
@JuliaEvseeva , LGTM

**Pull request merged**